src/game.cpp

```diff
@@ -12615,6 +12615,18 @@ void game::process_artifact( item &it, player &p )
                             it.charges++;
                         }
                         break;
+                    // Portals are energetic enough to charge the item.
+                    // Tears in reality are consumed too, but can't charge it.
+                    case ARTC_PORTAL:
+                        for( const tripoint &dest : m.points_in_radius( p.pos(), 1 ) ) {
+                            if( g->m.tr_at( dest ).loadid == tr_portal ) {
```

Comment:
Please access the same thing the same way for consistency. Here you use just `m` (and implicitly mean `this->m`), but in the line above you use `g->m`. Please remove the `g->` (the code is in a `game` member functions).